Skype/UCWA/user_ref.md

```diff
@@ -1,3 +1,6 @@
+---
+title: user as the entry point to the API using user credentials
```

Comment:
Change to "user reference for Skype for Business 2015" - longer than the others, because otherwise "user reference" looks like "a user reference" instead of "a reference for the resource that is named 'user'"

Reply:
done.